test/linalg/svd.jl

```diff
@@ -79,3 +79,25 @@ a2img  = randn(n,n)/2
         end
     end
 end
+
+@testset "Number input" begin
+    x, y = randn(2)
+    @test svdfact(x)   == svdfact(fill(x,1,1))
+    @test svdvals(x)   == first(svdvals(fill(x,1,1)))
+    @test svd(x)       == first.(svd(fill(x,1,1)))
+    @test svdfact(x,y) == svdfact(fill(x,1,1), fill(y,1,1))
+    @test svdvals(x,y) == first(svdvals(fill(x,1,1), fill(y,1,1)))
+    @test svd(x,y)     == first.(svd(fill(x,1,1), fill(y,1,1)))
+end
```

Comment:
We should choose a convention for spaces separating function arguments and use it consistently :).